packages/pegasus/add-params-to-evol-and-partons.patch

```diff
@@ -1,7 +1,7 @@
-diff --git a/Routines/initevol.f b/Routines/initevol.f
-index 43dcf7d..d993276 100644
---- a/Routines/initevol.f
-+++ b/Routines/initevol.f
+diff --git a/packages/pegasus/Pegasus/Routines/initevol.f b/packages/pegasus/Pegasus/Routines/initevol.f
+index 43dcf7d..9a531ac 100644
+--- a/packages/pegasus/Pegasus/Routines/initevol.f
++++ b/packages/pegasus/Pegasus/Routines/initevol.f
```

Comment:
You can see that here you are adding the `packages/pegasus/Pegasus/` relative path, but before it wasn't there.

If you run the installer with this, it will try to look for `packages/pegasus/Pegasus/` inside the `Pegasus` folder here.